docker/build_standalone-tcks.sh

```diff
@@ -78,6 +78,9 @@ echo "Git Revision: ${GIT_HASH}" >> ${WORKSPACE}/version.info
 echo "Git Branch: ${GIT_BRANCH}" >> ${WORKSPACE}/version.info
 echo "Build Date: ${BUILD_DATE}" >> ${WORKSPACE}/version.info
 
+#temporary fix to get build passing until we have glassfish with new api jars
+wget --progress=bar:force --no-cache https://repo1.maven.org/maven2/jakarta/platform/jakarta.jakartaee-api/9.0.0-RC1/jakarta.jakartaee-api-9.0.0-RC1.jar -O ${BASEDIR}/lib/jakartaee-api.jar
```

Comment:
Same here, it would be better if you move the jar to the GF, instead of keeping it in the lib folder.

Reply:
This is now addressed by downloading to $GF_HOME/glassfish5/glassfish/modules/jakartaee-api.jar, resolving this.